src/js/settings.js

```diff
@@ -506,7 +509,42 @@ function importSubscriptions(){
     })
   });
 }
+/**
+* Import NewPipe Channel Subscriptions
+* @return {Void}
+*/
+function importNewpipeSubscriptions(data){
 
+  progressView.seen = true;
+  progressView.width = 0;
+    
```

Comment:
Add a showToast call here using the same wording from the OPML import.

`showToast('Importing subscriptions, please wait.');`